src/agent/CommandLockAll.ts

```diff
@@ -19,9 +19,8 @@ class CommandLockAll extends CommandPolykey {
       const { default: PolykeyClient } = await import(
         'polykey/dist/PolykeyClient'
       );
-      const { default: WebSocketClient } = await import(
-        'polykey/dist/websockets/WebSocketClient'
-      );
+      const { WebSocketClient } = await import('@matrixai/ws');
+      const clientUtils = await import('polykey/dist/client/utils');
```

Comment:
Seems problematic...

Surely it's possible to expose the necessary abstractions directly from PK?